docs/how-to/h-enable-encryption.md

```diff
@@ -1,41 +1,54 @@
 # How to enable encryption
 
-Note: The TLS settings here are for self-signed-certificates which are not recommended for production clusters, the `tls-certificates-operator` charm offers a variety of configurations, read more on the TLS charm [here](https://charmhub.io/tls-certificates-operator)
+## Deploy a TLS Provider charm
 
-## Enable TLS
+To enable encryption, you should first deploy a TLS certificates Provider charm. The Kafka and ZooKeeper charms implements the Requirer side of the [`tls-certificates/v1`](https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/tls_certificates/v1/README.md) charm relation. 
+Therefore, any charm implementing the Provider side could be used. 
+
+One possible option, suitable for testing, could be to use the `self-signed-certificates`, although this setup is however not recommended for production clusters. 
+
+To deploy a `self-signed-certificates` charm:
 
 ```shell
 # deploy the TLS charm
-juju deploy tls-certificates-operator --channel=edge
+juju deploy self-signed-certificates --channel=edge
 # add the necessary configurations for TLS
-juju config tls-certificates-operator generate-self-signed-certificates="true" ca-common-name="Test CA"
-# to enable TLS relate the two applications
-juju relate tls-certificates-operator zookeeper
-juju relate tls-certificates-operator kafka
+juju config self-signed-certificates ca-common-name="Test CA"
 ```
 
+Please refer to [this post](https://charmhub.io/topics/security-with-x-509-certificates) for an overview of the TLS certificates Providers charms and some guidance on how to choose the right charm for your use-case. 
+
+## Enable TLS on Kafka and ZooKeeper
+
+```
+juju relate <tls-certificates> zookeeper
+juju relate <tls-certificates> kafka
```

Comment:
**todo:** This won't work. I believe that Kafka needs:

```
juju relate kafka:certificates <tls-certificates>
```

Because we also use that same interface for mTLS, so there's a conflict if we don't name it specifically.

Reply:
nice catch! Thanks